quantization/src/encoded_vectors_binary.rs

```diff
@@ -0,0 +1,188 @@
+use crate::encoded_vectors::validate_vector_parameters;
+use crate::utils::{transmute_from_u8_to_slice, transmute_to_u8_slice};
+use crate::{
+    EncodedStorage, EncodedStorageBuilder, EncodedVectors, EncodingError, VectorParameters,
+};
+use serde::{Deserialize, Serialize};
+use std::path::Path;
+
+type BitsStoreType = u128;
+
+const BITS_STORE_TYPE_SIZE: usize = std::mem::size_of::<BitsStoreType>() * 8;
```

Comment:
I guess it's better to use `std::mem::size_of::<usize>()`, because `8` should be different if usize is not 64bit value

Reply:
it's BITS not BYTES